tests/installation/partitioning/no_separate_home.pm

```diff
@@ -15,10 +15,11 @@
 use parent 'y2_installbase';
 use strict;
 use warnings FATAL => 'all';
+use testapi;
 
 sub run {
     my $partitioner = $testapi::distri->get_partitioner();
-    $partitioner->edit_proposal(has_separate_home => 0);
+    $partitioner->edit_proposal(has_separate_home => 0, multiple_disks => get_var('NUMDISKS', 0));
```

Comment:
As `multiple_disks` is boolean, I would propose to just convert the value to boolean using perl way rather than setting default value as `0`, which is wrong, as we have 1 disk in other cases.

```
$partitioner->edit_proposal(has_separate_home => 0, multiple_disks => !!get_var('NUMDISKS'));
```

Reply:
OK, thanks! The code is updated.